tsconfig.json

```diff
@@ -13,6 +13,7 @@
     "outDir": "./.temp",
     "alwaysStrict": true,
     "noImplicitReturns": true,
+    "noImplicitAny": false,
```

Comment:
what was the reason for this change?

Reply:
TS7022 error in the `createSubject()` method which was constantly driving me crazy:
![image](https://user-images.githubusercontent.com/5176060/83716762-81d83b80-a639-11ea-944b-c6bbb06be91c.png)